system/io.h

```diff
@@ -118,6 +119,105 @@ __OUTS(l)
     __inlc(port) : \
     __inl(port))
 
+#elif defined(__loongarch_lp64)
+
+#define LOONGSON_IO_PROT_BASE 0x0efdfc000000
+
+static __inline uint8_t __inb (int port)
```

Comment:
These `__inb` / `__inw` / `__inl` and `__outb` / `__outw` / `__outl` are relatively similar to each other. The existing implementations are macro-based anyway, so I'm not against using macros for these new implementations.

Reply:
Thanks.